src/Cache/Adapter/RedisCluster.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace Utopia\Cache\Adapter;
+
+use Exception;
+use RedisCluster as Client;
+use Throwable;
+use Utopia\Cache\Adapter;
+
+class RedisCluster implements Adapter
```

Comment:
```suggestion
class RedisCluster implements Redis
```
Lets extend the redis adapter and only override the necessary methods